### PR TITLE
Update observation reference range

### PR DESCRIPTION
**Pull request by lalitha94**
Added type field to Observation reference range.

Local Validation:
Screen Shot 2020-04-07 at 12 52 33 PM
Screen Shot 2020-04-07 at 11 53 46 AM

### CODE REVIEW

File: content/millennium/r4/diagnostic/observation.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ The following fields are returned if valued:
     * [low](https://hl7.org/fhir/R4/observation-definitions.html#Observation.referenceRange.low){:target="_blank"}
     * [high](https://hl7.org/fhir/R4/observation-definitions.html#Observation.referenceRange.high){:target="_blank"}
     * [text](https://hl7.org/fhir/R4/observation-definitions.html#Observation.referenceRange.text){:target="_blank"}
+    * [type](https://hl7.org/fhir/R4/observation-definitions.html#Observation.referenceRange.type){:target="_blank"}
```
Comment (SA030800) [inline content/millennium/r4/diagnostic/observation.md:35]: according to the FHIR spec: http://hl7.org/fhir/r4/observation.html
**type** should come before **text**

Comment (lalitha94) [reply content/millennium/r4/diagnostic/observation.md:34]: https://github.com/cerner/fhir.cerner.com/pull/373/commits/102c1720e4f465083aed0e3cd865e60504cd441a

**Comment by SA030800** [comment]
would you mind updating the screenshot as well

**Comment by lalitha94** [comment]
> would you mind updating the screenshot as well

Updated

**Comment by MaxPhilips** [comment]
Screen Shot 2020-04-07 at 5 30 11 PM
validated deployment

**Pull request merged**